lightning/src/util/ser_macros.rs

```diff
@@ -163,6 +169,10 @@ macro_rules! decode_tlv {
 	($reader: expr, $field: ident, ignorable) => {{
 		$field = ser::MaybeReadable::read(&mut $reader)?;
 	}};
+	// Read a ser::LengthReadableArgs
```

Comment:
Probably can drop the comment now that this is compatible with any sort of `Readable`.